### DISCUSSION TITLE
Simpel interface to elspotpris

### DISCUSSION THREAD

**Discussion by telecpu** [rndfm/elspotpris#24] [Ideas]
Hi
I use lot of time to look after best price window, to charge my car. Why not create a simpel file to download automatic, with date,hour and price in one line each time.
Point is to start to charce, at  right time, and maybe stop/pause.
Format ex: 202207201200:15
                 YYYYMMDDTT
If it was ready each day 15:00 then user can download, and use it.
Welcome to discuss

**Comment by rndfm** [comment]
I have some plans to make an API for elspotpris. But it should provide more functionality then the existing providers.
Like get hours to run power hungry devices. Where one could provide the the needed power for a period and the API would return the best period to draw the load.

Any reason for the very specific format?
You can get the prices from the Energidataservice or nordpool directly. Both return Json. But they are not able to include taxes or prices for a specific power company.

**Comment by rndfm** [comment]
What kind of automation are you running to control the charging of the car?

**Comment by telecpu** [comment]
hi mdfm. I use a PLC to run my house. So I think to convert it to modbus, and get in that way in PLC. At the same time make decision logic to find best period. In my case i know i need 3,5 hour to charge car, but friday if no need to use the car then wait to maybe saturday. (take the chance). Now i need to setup timer, and sometimes you forget, and you then charge when it is most exspensive. KimHent Outlook til Android
